### PR TITLE
 Making sure that the test suite's OpenShift client trusts certificates provided by API server explicitely.

### PR DESCRIPTION
**Pull request by mjurc**

### CODE REVIEW

File: common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java [hunk 1]
```diff
@@ -2,13 +2,18 @@
...
+import io.fabric8.openshift.client.OpenShiftConfigBuilder;
 import org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource;
 
 final class OpenShiftClientResource implements CloseableResource {
     final OpenShiftClient client;
 
     static OpenShiftClientResource createDefault() {
-        return new OpenShiftClientResource(new DefaultOpenShiftClient());
+        OpenShiftConfig config = new OpenShiftConfigBuilder()
+                .withTrustCerts(true)
```
Comment (rsvoboda) [inline common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java:14]: So this is equivalent to `quarkus.kubernetes-client.trust-certs=true`, right?

Comment (Ladicek) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java:14]: Yes.

Comment (mjurc) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java:14]: Indeed:

https://github.com/quarkusio/quarkus/blob/53d41bd92b127d3ecc76f9678414343696ded7c8/extensions/kubernetes-client/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientUtils.java#L53

Comment (Ladicek) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java:14]: I mean, the commit message is misleading :-)

Comment (rsvoboda) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java:14]: Should @mjurc amend it?

Comment (mjurc) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java:14]: I've updated it :)

**Pull request merged**